### ISSUE TITLE
[BUG] Forbidden characters are allowed in create entities fields - NGSI v2

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#1093]
Failing Fields:

* "type"
* "id"
* "attribute name"
* "attribute value"
* "attribute type"
* "attribute metadata name"
* "attribute metadata value"
* "attribute metadata type"

### Request:
```
http://<host>:<port>/v2/entities
Fiware-Service: sdsd_1
Fiware-ServicePath: /service
Accept: application/json

- payload to type field:
         {"temperature_0": "34", "type": "house;flat", "id": "room2_0"}
- payload toid field:
         {"temperature_0": "34", "id": "house;flat", "type": "room"}
- payload to attribute name field:
         {"house;flat": "34", "type": "room", "id": "room2_0"}
- payload to attribute value field:
         {"temperature": "house;flat", "type": "room", "id": "room2_0"}
- payload to attribute type field:
         {"temperature": {"type": "house;flat" "value": " 34"}, "type": "room", "id": "room2_0"}
- payload to attribute metadata name field:
         {"temperature": {"type": "data", "value": " 34", "house;flat": "danger"}, "type": "room", "id": "room2_0"}
- payload to attribute metadata value field:
         {"temperature": {"type": "data", "value": " 34", "meta": "house;flat"}, "type": "room", "id": "room2_0"}
- payload to attribute metadata type field:
         {"temperature": {"type": "data", "value": " 34", "meta": {"value":"45","type": "house;flat"}}, "type": "room", "id": "room2_0"}
```

### Response:
```
status code: 201 Created
```

### Examples in mongo
```
{ "_id" : { "id" : "room2_0", "type" : "house<flat>", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437571371, "modDate" : 1437571371, "value" : "34" } }, "creDate" : 1437571371, "modDate" : 1437571371 }
{ "_id" : { "id" : "room2_0", "type" : "house=flat", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437571372, "modDate" : 1437571372, "value" : "34" } }, "creDate" : 1437571372, "modDate" : 1437571372 }
{ "_id" : { "id" : "room2_0", "type" : "house\"flat\"", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437571372, "modDate" : 1437571372, "value" : "34" } }, "creDate" : 1437571372, "modDate" : 1437571372 }
{ "_id" : { "id" : "room2_0", "type" : "house'flat'", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437571372, "modDate" : 1437571372, "value" : "34" } }, "creDate" : 1437571372, "modDate" : 1437571372 }
{ "_id" : { "id" : "room2_0", "type" : "house;flat", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437571373, "modDate" : 1437571373, "value" : "34" } }, "creDate" : 1437571373, "modDate" : 1437571373 }
{ "_id" : { "id" : "room2_0", "type" : "house,flat", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437571373, "modDate" : 1437571373, "value" : "34" } }, "creDate" : 1437571373, "modDate" : 1437571373 }
{ "_id" : { "id" : "room2_0", "type" : "house(flat)", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437571373, "modDate" : 1437571373, "value" : "34" } }, "creDate" : 1437571373, "modDate" : 1437571373 }
```
```
{ "_id" : { "id" : "house<flat>_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437738805, "modDate" : 1437738805, "value" : "34" } }, "creDate" : 1437738805, "modDate" : 1437738805 }
{ "_id" : { "id" : "house=flat_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437738805, "modDate" : 1437738805, "value" : "34" } }, "creDate" : 1437738805, "modDate" : 1437738805 }
{ "_id" : { "id" : "house\"flat\"_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437738805, "modDate" : 1437738805, "value" : "34" } }, "creDate" : 1437738805, "modDate" : 1437738805 }
{ "_id" : { "id" : "house'flat'_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437738806, "modDate" : 1437738806, "value" : "34" } }, "creDate" : 1437738806, "modDate" : 1437738806 }
{ "_id" : { "id" : "house;flat_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437738806, "modDate" : 1437738806, "value" : "34" } }, "creDate" : 1437738806, "modDate" : 1437738806 }
{ "_id" : { "id" : "house,flat_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437738806, "modDate" : 1437738806, "value" : "34" } }, "creDate" : 1437738806, "modDate" : 1437738806 }
{ "_id" : { "id" : "house(flat)_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437738807, "modDate" : 1437738807, "value" : "34" } }, "creDate" : 1437738807, "modDate" : 1437738807 }
```
```
{ "_id" : { "id" : "room_2_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "house<flat>_0" ], "attrs" : { "house<flat>_0" : { "type" : "", "creDate" : 1437739735, "modDate" : 1437739735, "value" : "34" } }, "creDate" : 1437739735, "modDate" : 1437739735 }
{ "_id" : { "id" : "room_3_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "house=flat_0" ], "attrs" : { "house=flat_0" : { "type" : "", "creDate" : 1437739736, "modDate" : 1437739736, "value" : "34" } }, "creDate" : 1437739736, "modDate" : 1437739736 }
{ "_id" : { "id" : "room_4_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "house\"flat\"_0" ], "attrs" : { "house"flat"_0" : { "type" : "", "creDate" : 1437739736, "modDate" : 1437739736, "value" : "34" } }, "creDate" : 1437739736, "modDate" : 1437739736 }
{ "_id" : { "id" : "room_5_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "house'flat'_0" ], "attrs" : { "house'flat'_0" : { "type" : "", "creDate" : 1437739737, "modDate" : 1437739737, "value" : "34" } }, "creDate" : 1437739737, "modDate" : 1437739737 }
{ "_id" : { "id" : "room_6_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "house;flat_0" ], "attrs" : { "house;flat_0" : { "type" : "", "creDate" : 1437739737, "modDate" : 1437739737, "value" : "34" } }, "creDate" : 1437739737, "modDate" : 1437739737 }
{ "_id" : { "id" : "room_7_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "house,flat_0" ], "attrs" : { "house,flat_0" : { "type" : "", "creDate" : 1437739737, "modDate" : 1437739737, "value" : "34" } }, "creDate" : 1437739737, "modDate" : 1437739737 }
{ "_id" : { "id" : "room_8_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "house(flat)_0" ], "attrs" : { "house(flat)_0" : { "type" : "", "creDate" : 1437739737, "modDate" : 1437739737, "value" : "34" } }, "creDate" : 1437739737, "modDate" : 1437739737 }
{ "_id" : { "id" : "room_9_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "%~#_0" ], "attrs" : { "%~#_0" : { "type" : "", "creDate" : 1437739738, "modDate" : 1437739738, "value" : "34" } }, "creDate" : 1437739738, "modDate" : 1437739738 }
```
```
{ "_id" : { "id" : "room2_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "value" : "%~#", "type" : "", "creDate" : 1437740264, "modDate" : 1437740266 } }, "creDate" : 1437740264, "modDate" : 1437740266 }
{ "_id" : { "id" : "room_2_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437740873, "modDate" : 1437740873, "value" : "house<flat>" } }, "creDate" : 1437740873, "modDate" : 1437740873 }
{ "_id" : { "id" : "room_3_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437740873, "modDate" : 1437740873, "value" : "house=flat" } }, "creDate" : 1437740873, "modDate" : 1437740873 }
{ "_id" : { "id" : "room_4_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437740873, "modDate" : 1437740873, "value" : "house\"flat\"" } }, "creDate" : 1437740873, "modDate" : 1437740873 }
{ "_id" : { "id" : "room_5_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437740874, "modDate" : 1437740874, "value" : "house'flat'" } }, "creDate" : 1437740874, "modDate" : 1437740874 }
{ "_id" : { "id" : "room_6_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437740874, "modDate" : 1437740874, "value" : "house;flat" } }, "creDate" : 1437740874, "modDate" : 1437740874 }
{ "_id" : { "id" : "room_7_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437740874, "modDate" : 1437740874, "value" : "house,flat" } }, "creDate" : 1437740874, "modDate" : 1437740874 }
{ "_id" : { "id" : "room_8_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437740875, "modDate" : 1437740875, "value" : "house(flat)" } }, "creDate" : 1437740875, "modDate" : 1437740875 }
{ "_id" : { "id" : "room_9_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437740875, "modDate" : 1437740875, "value" : "%~#" } }, "creDate" : 1437740875, "modDate" : 1437740875 }
```
```
{ "_id" : { "id" : "room_2_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "house<flat>", "creDate" : 1437741246, "modDate" : 1437741246, "value" : "34" } }, "creDate" : 1437741246, "modDate" : 1437741246 }
{ "_id" : { "id" : "room_3_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "house=flat", "creDate" : 1437741246, "modDate" : 1437741246, "value" : "34" } }, "creDate" : 1437741246, "modDate" : 1437741246 }
{ "_id" : { "id" : "room_4_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "house\"flat\"", "creDate" : 1437741247, "modDate" : 1437741247, "value" : "34" } }, "creDate" : 1437741247, "modDate" : 1437741247 }
{ "_id" : { "id" : "room_5_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "house'flat'", "creDate" : 1437741247, "modDate" : 1437741247, "value" : "34" } }, "creDate" : 1437741247, "modDate" : 1437741247 }
{ "_id" : { "id" : "room_6_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "house;flat", "creDate" : 1437741247, "modDate" : 1437741247, "value" : "34" } }, "creDate" : 1437741247, "modDate" : 1437741247 }
{ "_id" : { "id" : "room_7_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "house,flat", "creDate" : 1437741248, "modDate" : 1437741248, "value" : "34" } }, "creDate" : 1437741248, "modDate" : 1437741248 }
{ "_id" : { "id" : "room_8_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "house(flat)", "creDate" : 1437741248, "modDate" : 1437741248, "value" : "34" } }, "creDate" : 1437741248, "modDate" : 1437741248 }
{ "_id" : { "id" : "room_9_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "%~#", "creDate" : 1437741248, "modDate" : 1437741248, "value" : "34" } }, "creDate" : 1437741248, "modDate" : 1437741248 }
```
```
{ "_id" : { "id" : "room_1_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742459, "modDate" : 1437742459, "value" : "34", "md" : [ { "name" : "_0", "value" : "0I2lE" } ] } }, "creDate" : 1437742459, "modDate" : 1437742459 }
{ "_id" : { "id" : "room_2_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742459, "modDate" : 1437742459, "value" : "34", "md" : [ { "name" : "house<flat>_0", "value" : "mOUrO" } ] } }, "creDate" : 1437742459, "modDate" : 1437742459 }
{ "_id" : { "id" : "room_3_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742459, "modDate" : 1437742459, "value" : "34", "md" : [ { "name" : "house=flat_0", "value" : "DRMLL" } ] } }, "creDate" : 1437742459, "modDate" : 1437742459 }
{ "_id" : { "id" : "room_4_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742460, "modDate" : 1437742460, "value" : "34", "md" : [ { "name" : "house\"flat\"_0", "value" : "SosWD" } ] } }, "creDate" : 1437742460, "modDate" : 1437742460 }
{ "_id" : { "id" : "room_5_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742460, "modDate" : 1437742460, "value" : "34", "md" : [ { "name" : "house'flat'_0", "value" : "o3FWe" } ] } }, "creDate" : 1437742460, "modDate" : 1437742460 }
{ "_id" : { "id" : "room_6_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742460, "modDate" : 1437742460, "value" : "34", "md" : [ { "name" : "house;flat_0", "value" : "3iKE6" } ] } }, "creDate" : 1437742460, "modDate" : 1437742460 }
{ "_id" : { "id" : "room_7_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742461, "modDate" : 1437742461, "value" : "34", "md" : [ { "name" : "house,flat_0", "value" : "6Yf2K" } ] } }, "creDate" : 1437742461, "modDate" : 1437742461 }
{ "_id" : { "id" : "room_8_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742461, "modDate" : 1437742461, "value" : "34", "md" : [ { "name" : "house(flat)_0", "value" : "KiKPt" } ] } }, "creDate" : 1437742461, "modDate" : 1437742461 }
{ "_id" : { "id" : "room_9_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742461, "modDate" : 1437742461, "value" : "34", "md" : [ { "name" : "%~#_0", "value" : "06E3S" } ] } }, "creDate" : 1437742461, "modDate" : 1437742461 }
```
```
{ "_id" : { "id" : "room_2_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742459, "modDate" : 1437742459, "value" : "34", "md" : [ { "name" : "house<flat>_0", "value" : "mOUrO" } ] } }, "creDate" : 1437742459, "modDate" : 1437742459 }
{ "_id" : { "id" : "room_3_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742459, "modDate" : 1437742459, "value" : "34", "md" : [ { "name" : "house=flat_0", "value" : "DRMLL" } ] } }, "creDate" : 1437742459, "modDate" : 1437742459 }
{ "_id" : { "id" : "room_4_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742460, "modDate" : 1437742460, "value" : "34", "md" : [ { "name" : "house\"flat\"_0", "value" : "SosWD" } ] } }, "creDate" : 1437742460, "modDate" : 1437742460 }
{ "_id" : { "id" : "room_5_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742460, "modDate" : 1437742460, "value" : "34", "md" : [ { "name" : "house'flat'_0", "value" : "o3FWe" } ] } }, "creDate" : 1437742460, "modDate" : 1437742460 }
{ "_id" : { "id" : "room_6_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742460, "modDate" : 1437742460, "value" : "34", "md" : [ { "name" : "house;flat_0", "value" : "3iKE6" } ] } }, "creDate" : 1437742460, "modDate" : 1437742460 }
{ "_id" : { "id" : "room_7_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742461, "modDate" : 1437742461, "value" : "34", "md" : [ { "name" : "house,flat_0", "value" : "6Yf2K" } ] } }, "creDate" : 1437742461, "modDate" : 1437742461 }
{ "_id" : { "id" : "room_8_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742461, "modDate" : 1437742461, "value" : "34", "md" : [ { "name" : "house(flat)_0", "value" : "KiKPt" } ] } }, "creDate" : 1437742461, "modDate" : 1437742461 }
{ "_id" : { "id" : "room_9_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437742461, "modDate" : 1437742461, "value" : "34", "md" : [ { "name" : "%~#_0", "value" : "06E3S" } ] } }, "creDate" : 1437742461, "modDate" : 1437742461 }
```
```
{ "_id" : { "id" : "room_2_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437745030, "modDate" : 1437745030, "value" : "34", "md" : [ { "name" : "q8EOe_0", "value" : "house<flat>" } ] } }, "creDate" : 1437745030, "modDate" : 1437745030 }
{ "_id" : { "id" : "room_3_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437745030, "modDate" : 1437745030, "value" : "34", "md" : [ { "name" : "HBeHO_0", "value" : "house=flat" } ] } }, "creDate" : 1437745030, "modDate" : 1437745030 }
{ "_id" : { "id" : "room_4_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437745030, "modDate" : 1437745030, "value" : "34", "md" : [ { "name" : "WGj2I_0", "value" : "house\"flat\"" } ] } }, "creDate" : 1437745030, "modDate" : 1437745030 }
{ "_id" : { "id" : "room_5_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437745031, "modDate" : 1437745031, "value" : "34", "md" : [ { "name" : "1NCnE_0", "value" : "house'flat'" } ] } }, "creDate" : 1437745031, "modDate" : 1437745031 }
{ "_id" : { "id" : "room_6_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437745031, "modDate" : 1437745031, "value" : "34", "md" : [ { "name" : "2ybBT_0", "value" : "house;flat" } ] } }, "creDate" : 1437745031, "modDate" : 1437745031 }
{ "_id" : { "id" : "room_7_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437745031, "modDate" : 1437745031, "value" : "34", "md" : [ { "name" : "f5Ijj_0", "value" : "house,flat" } ] } }, "creDate" : 1437745031, "modDate" : 1437745031 }
{ "_id" : { "id" : "room_8_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437745032, "modDate" : 1437745032, "value" : "34", "md" : [ { "name" : "t61hW_0", "value" : "house(flat)" } ] } }, "creDate" : 1437745032, "modDate" : 1437745032 }
{ "_id" : { "id" : "room_9_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437745032, "modDate" : 1437745032, "value" : "34", "md" : [ { "name" : "fc2jk_0", "value" : "%~#" } ] } }, "creDate" : 1437745032, "modDate" : 1437745032 
```
```
{ "_id" : { "id" : "room_2_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437746146, "modDate" : 1437746146, "value" : "34", "md" : [ { "name" : "RMkwL_0", "type" : "house<flat>", "value" : "PtTjgk" } ] } }, "creDate" : 1437746146, "modDate" : 1437746146 }
{ "_id" : { "id" : "room_3_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437746146, "modDate" : 1437746146, "value" : "34", "md" : [ { "name" : "4KBwq_0", "type" : "house=flat", "value" : "TOufl9" } ] } }, "creDate" : 1437746146, "modDate" : 1437746146 }
{ "_id" : { "id" : "room_4_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437746146, "modDate" : 1437746146, "value" : "34", "md" : [ { "name" : "FWgtn_0", "type" : "house\"flat\"", "value" : "0MDgXN" } ] } }, "creDate" : 1437746146, "modDate" : 1437746146 }
{ "_id" : { "id" : "room_5_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437746147, "modDate" : 1437746147, "value" : "34", "md" : [ { "name" : "a2uxH_0", "type" : "house'flat'", "value" : "u5eTUB" } ] } }, "creDate" : 1437746147, "modDate" : 1437746147 }
{ "_id" : { "id" : "room_6_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437746147, "modDate" : 1437746147, "value" : "34", "md" : [ { "name" : "iWFPQ_0", "type" : "house;flat", "value" : "PGaHKp" } ] } }, "creDate" : 1437746147, "modDate" : 1437746147 }
{ "_id" : { "id" : "room_7_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437746148, "modDate" : 1437746148, "value" : "34", "md" : [ { "name" : "6MuRZ_0", "type" : "house,flat", "value" : "e930Tv" } ] } }, "creDate" : 1437746148, "modDate" : 1437746148 }
{ "_id" : { "id" : "room_8_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437746148, "modDate" : 1437746148, "value" : "34", "md" : [ { "name" : "7DiWE_0", "type" : "house(flat)", "value" : "DpxbUK" } ] } }, "creDate" : 1437746148, "modDate" : 1437746148 }
{ "_id" : { "id" : "room_9_0", "type" : "room", "servicePath" : "/test" }, "attrNames" : [ "temperature_0" ], "attrs" : { "temperature_0" : { "type" : "", "creDate" : 1437746148, "modDate" : 1437746148, "value" : "34", "md" : [ { "name" : "3LgKJ_0", "type" : "%~#", "value" : "iOYljw" } ] } }, "creDate" : 1437746148, "modDate" : 1437746148 }

```

**Comment by fgalan** [comment]
Fixed in PR #1143. Waiting for @iariasleon to check that the code is now ok and close the issue.

**Comment by iariasleon** [comment]
LGTM
